ape-config.yaml

```
@@ -0,0 +1,3 @@
+ethereum:
+  mainnet:
+    default_provider: infura
```

this is more about how it gets deployed, I wouldn't specify this config, but I would add `ens` to the plugins to install e.g.
```suggestion
plugins:
- name: ens
```